format/FormatHDF5SaclaRayonix.py

```diff
@@ -65,7 +63,11 @@ def _start(self):
         h5_handle.close()
 
     def read_metadata(self):
-        pass
+        import h5py
+
+        h5_handle = h5py.File(self.image_filename, "r")
+        self.pixelsize_in_um = h5_handle["metadata"]["pixelsize_in_um"].value
```

Comment:
I believe `.value` will throw a `DeprecationWarning` and the recommended method to access the value is to replace `.value` by `[()]`.